### ISSUE TITLE
Contact Form: Keyboard navigation fails on "Email Notifications" tab

### ISSUE THREAD

**Issue by kraftbj** [Automattic/jetpack#663]
To duplicate:
1. Add a new contact form via the modal button in the post editor.
2. Choose the second tab at the top, E-Mail Notifications
3. Change/add a custom notification e-mail address or subject.
4. Press Enter to submit the form.

Expected:
It accepts the data from that tab, returning you to the Form Builder tab.

Actual:
It submits the Form Builder form, adding the shortcode to the post editor. Since the E-Mail Notifications form was never submitted, the e-mail address/subject changes are lost.

**Comment by jeherve** [comment]
Closing this for now because of the lack of activity on this. We can always reopen in the future if needed.
